PyInstaller/depend/dylib.py

```diff
@@ -200,6 +200,7 @@
     r'libnss_nis.*\.so(\..*)?',
     r'libnss_nisplus.*\.so(\..*)?',
     r'libresolv\.so(\..*)?',
+    r'libsocket\.so(\..*)?',
```

Comment:
Can we limit this to solaris? There's more than one `libsocket` out there.

Reply:
Right I see what you mean and that makes sense. I've made the necessary change